### PR TITLE
docs(samples): add code samples

### PR DESCRIPTION
**Pull request by irataxy**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [X] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-video-live-stream/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [X] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [X] Appropriate docs were updated (if necessary)

Fixes b:219762506

### CODE REVIEW

**Comment by parthea** [comment]
Wait for cl/444382370

**Comment by parthea** [comment]
Googlers see b/203559874

File: samples/snippets/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        $ pip install -r requirements.txt
+
+## The client library
+
+This sample uses the
+[Google Cloud Client Library for Python](https://googlecloudplatform.github.io/google-cloud-python/).
+You can read the documentation for more details on API usage and use GitHub
+to
+[browse the source](https://github.com/GoogleCloudPlatform/google-cloud-python)
+and [report issues](https://github.com/GoogleCloudPlatform/google-cloud-python/issues).
```
Comment (parthea) [inline samples/snippets/README.md:47]: Please delete this section or update the links to point to this repo (and corresponding issues section for this repo  https://github.com/googleapis/python-video-live-stream/issues/new/)

**Review by parthea — CHANGES_REQUESTED** [review]
Added minor observations

File: samples/snippets/README.md (hunk 1 above)
Comment (irataxy) [reply samples/snippets/README.md]: Removed section

File: samples/snippets/create_channel.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+import argparse
+
+from google.cloud.video import live_stream_v1
+from google.cloud.video.live_stream_v1.services.livestream_service import (
+    LivestreamServiceClient,
+)
+from google.protobuf import duration_pb2 as duration
+
+
+def create_channel(project_number, location, channel_id, input_id, output_uri):
```
[the remark this answers is not in the document]
Comment (irataxy) [reply samples/snippets/create_channel.py]: Added to all methods

**Review by parthea — APPROVED** [review]

**Pull request merged**